frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js

```diff
@@ -162,7 +162,12 @@ export default function InteractiveWaveformModal({ selectedJob, setOpenInteracti
   const [undoing, setUndoing] = useState(false);
   const [peakValleyWindows, setPeakValleyWindows] = useState({});
   const [duplicatesPresent, setDuplicatesPresent] = useState(false);
+  const [duplicateModalOpen, setDuplicateModalOpen] = useState(false);
 
+  const handleDuplicatesModalClose = (choice) => {
+    setDuplicateModalOpen(false);
+    choice === 1 ? postNewJob() : null;
```

Comment:
I would argue that using the ternary operator like this to conditionally run code is bad style. The most common use case you see for it is to conditionally assign a value to a variable, so when that is not the case the intent becomes unclear and makes the code less readable. Also, in this instance there's nothing to do if `choice` is falsey, but you are still forced to put a `null` in the statement which looks clunky.

Using a regular old `if` statement makes the intent immediately clear, and if you really want to keep it as a one liner you could do
```
if (choice) postNewJob();
```
Although I would also argue that
```
if (choice) {
  postNewJob();
}
```
is the most readable option. Also still think that `choice` should be renamed here to something that makes it obvious what action the user chose, like `runAnalysis`